presto-docs/src/main/sphinx/admin/tuning.rst

```diff
@@ -10,29 +10,392 @@ Config Properties
++   ``query.schedule-split-batch-size``    +       ``Integer`` (at least ``1``)       +              ``1000``              +The size of single data chunk expressed in rows that   +
++                                          +                                          +                                    +will be processed as single split. Higher value may be +
++                                          +                                          +                                    +used if system works in reliable environment and there +
++                                          +                                          +                                    +the responsiveness is less important then average      +
++                                          +                                          +                                    +answer time. Decreasing this value may have a positive +
++                                          +                                          +                                    +effect if there are lots of nodes in system and        +
++                                          +                                          +                                    +calculations are relatively heavy for each of rows.    +
++                                          +                                          +                                    +Other scenario may be if there are many nodes with     +
++                                          +                                          +                                    +poor stability - lowering this number will allow to    +
++                                          +                                          +                                    +react faster and for that reason the lost computation  +
++                                          +                                          +                                    +time will be potentially lower.                        +
++------------------------------------------+------------------------------------------+------------------------------------+-------------------------------------------------------+
+
+
+Session properties
+------------------
+
++-------------------------------------+------------------------------------------+--------------------------------------------+----------------------------------------------------+
++            property name            +                   type                   +               default value                +short description                                   +
++=====================================+==========================================+============================================+====================================================+
++      ``task_join_concurrency``      +               ``Integer``                +     ``task.join-concurrency`` (``1``)      +**Experimental.** Default number of local parallel  +
```

Comment:
It's annotated as experimental in code. As documentation of this is poor I have no idea. Based on my guess I assume that it's property that can be dropped anytime breaking backward-compatibility for its users.